src/App.js

```diff
@@ -1,12 +1,33 @@
 import React from 'react';
+import CardUi from './components/CardUi'
```

Comment:
Should make the imports absolute to src folder via webpack or jsconfig. Will do that and update imports once the PR is merged.